docs/rings.md

```diff
@@ -0,0 +1,357 @@
+# Walkthrough: Rings deployment and management with automation
+
+If you're new to the concept of Rings, it is suggested you read through the [spk Rings guide](https://github.com/CatalystCode/spk/blob/master/guides/rings-101.md#what-are-deployment-rings). In short, in the context of Kubernetes, Rings are a way to encapsulate and deploy multiple environments in a single cluster.
+
+This walkthrough will guide you through enabling Ringed deployments of your project with automation through the `spk` tool. We will be utilizing the azure-vote example that was set up during the [Service Management Walkthrough](./docs/../services.md).
+
+## Prerequisites
+1. Completion of the [First Workload guide](./docs/firstWorkload/README.md) to setup an AKS cluster configured with flux.
+2. Completion of the [GitOps Pipeline Walkthrough](./docs/hld-to-manifest.md) to set up required GitOps workflow repositories and pipelines.
+3. Completion of the [Service Management Walkthrough](./docs/../services.md)
+4. `spk` version `v0.5.9` (actual version tbd once the lastest release is out.) or later. [SPK Releases](https://github.com/CatalystCode/spk/releases)
+
+## Enabling Ingress Routes
+[Traefik2](https://github.com/containous/traefik) is the default LoadBalancer utilized to route ingress traffic to a specific ring.
+
+When setting up the HLD (high-level-definition) for the first time with `spk hld init`, Traefik2 is included as the initial component. If it was not included or removed as a component, please clone the HLD repository and add Traefik as a component:
+
+```sh
+git clone git@ssh.dev.azure.com:v3/mtarng/my-spk-project/fabrikam-hld
+...
+cd fabrikam-hld
+fab add traefik2 --source https://github.com/microsoft/fabrikate-definitions.git --path definitions/traefik2
+git add -A
+git commit -m 'Adding Traefik2 component'
+```
+
+Additionally, if `disableRouteScaffold` was added to the project `bedrock.yaml` for the azure-vote service during the Service Management walkthrough, go ahead and remove it.
+
+Your project `bedrock.yaml` should look like this when updated:
+
+```yaml
+rings:
+  master:
+    isDefault: true
+services:
+  ./azure-vote:
+    disableRouteScaffold: true
```

Comment:
Shouldn't this have been removed per the instructions above?

Reply:
Good eye, fixed.